<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>474</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>QRenamer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pics.qrc">
    <normaloff>:/pics/macro_names.png</normaloff>:/pics/macro_names.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="policyTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>8</number>
      </property>
      <widget class="QWidget" name="prependZeroNumberTab">
       <attribute name="title">
        <string>Prepend zero on number</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="numberComboBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Number length</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="numberSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="positionSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="prependWordsTab">
       <attribute name="title">
        <string>Prepend words</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Prepend</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="prependLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="appendWordsTab">
       <attribute name="title">
        <string>Append words</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Append</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="appendLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="insertTab">
       <attribute name="title">
        <string>Insert words</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Insert at</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="insertAtSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Insert words</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="insertWordsLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="removeTab">
       <attribute name="title">
        <string>Remove words</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Remove at</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="removeAtSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="removeRanageSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="replaceWordsTab">
       <attribute name="title">
        <string>Replace words</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="replaceFromLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="replaceTolineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="regexTab">
       <attribute name="title">
        <string>Regex</string>
       </attribute>
       <attribute name="toolTip">
        <string>Replace names by regular expression</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Match</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Match</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="regexMatchlineEdit">
             <property name="text">
              <string>\d+</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="icaseCheckBox">
             <property name="text">
              <string>Case insensitive</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Replace</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Replace</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="regexReplacelineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>$#</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="numRadioButton">
             <property name="text">
              <string>Numbering</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QToolButton" name="numStepToolButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Adjust the numbering step</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../pics.qrc">
               <normaloff>:/pics/num_step.png</normaloff>:/pics/num_step.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="replaceSuffixTab">
       <attribute name="title">
        <string>Replace suffix</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="replaceSuffixFromLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="replaceSuffixTolineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Change name with increase num</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Change to file name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditChangeToFileName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Zero fill</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxZeroFill"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="fileView">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::CopyAction</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="renameFileButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pics.qrc">
        <normaloff>:/pics/accept.png</normaloff>:/pics/accept.png</iconset>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="renameProgressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFiles"/>
   <addaction name="actionDeleteFiles"/>
   <addaction name="actionAddFolder"/>
   <addaction name="actionRemoveAll"/>
   <addaction name="actionAddFolderRecursive"/>
   <addaction name="actionSpanHeader"/>
   <addaction name="actionRevert"/>
   <addaction name="actionFilter"/>
   <addaction name="actionRemovefilter"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowPath"/>
    <addaction name="actionShowSuffix"/>
    <addaction name="actionSpanHeader"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionDeleteFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="actionRemoveAll"/>
    <addaction name="actionAddFolderRecursive"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionRevert"/>
    <addaction name="actionFilter"/>
    <addaction name="actionRemovefilter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionIcons"/>
    <addaction name="actionReference"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionAddFiles">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/pics/add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
   <property name="toolTip">
    <string>Add files</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionDeleteFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/delete.png</normaloff>:/pics/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete files</string>
   </property>
   <property name="toolTip">
    <string>Delete selected files</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/folder.png</normaloff>:/pics/folder.png</iconset>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
   <property name="toolTip">
    <string>Add the files under specific folder</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/clipboard_empty.png</normaloff>:/pics/clipboard_empty.png</iconset>
   </property>
   <property name="text">
    <string>Remove all</string>
   </property>
   <property name="toolTip">
    <string>Remove all of the files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAddFolderRecursive">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/folders_explorer.png</normaloff>:/pics/folders_explorer.png</iconset>
   </property>
   <property name="text">
    <string>Add folder recursive</string>
   </property>
   <property name="toolTip">
    <string>Add the files of the folder recursively</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShowPath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show path</string>
   </property>
   <property name="toolTip">
    <string>Show the path of the file</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionShowSuffix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show suffix</string>
   </property>
   <property name="toolTip">
    <string>Show the suffix of the path</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSpanHeader">
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/spanHeader.png</normaloff>:/pics/spanHeader.png</iconset>
   </property>
   <property name="text">
    <string>Span header</string>
   </property>
   <property name="toolTip">
    <string>Span the header of the view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="Revert" resource="../pics.qrc">
     <normaloff>:/pics/revert.png</normaloff>:/pics/revert.png</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert the file you changed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="Filter" resource="../pics.qrc">
     <normaloff>:/pics/filter.png</normaloff>:/pics/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Filter the contents</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/help.png</normaloff>:/pics/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionRemovefilter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/filter_clear.png</normaloff>:/pics/filter_clear.png</iconset>
   </property>
   <property name="text">
    <string>Removefilter</string>
   </property>
   <property name="toolTip">
    <string>Remove filter</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionIcons">
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/app_icons.png</normaloff>:/pics/app_icons.png</iconset>
   </property>
   <property name="text">
    <string>Icons</string>
   </property>
   <property name="toolTip">
    <string>The link of the icons</string>
   </property>
  </action>
  <action name="actionReference">
   <property name="icon">
    <iconset resource="../pics.qrc">
     <normaloff>:/pics/cross_reference.png</normaloff>:/pics/cross_reference.png</iconset>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../pics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>fileView</receiver>
   <slot>resizeColumnsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
